sdk/identity/identity/tests.yml

```diff
@@ -3,6 +3,22 @@ trigger: none
 extends:
     template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
     parameters:
+      PreSteps:
```

Comment:
Shamelessly stolen from dotnet https://github.com/Azure/azure-sdk-for-net/blob/d5f3c397728a00bc1f2c2ae886991fa22e38ed16/sdk/identity/tests.yml#L5-L40